src/Questions.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+const Question = (props) => {
+  const allAnswer = [
+    ...props.value.incorrect_answers,
+    props.value.correct_answer,
+  ];
+  const [selected, setSelected] = React.useState(null);
+  let styleQue = props.checkAns
+    ? selected === props.value.correct_answer
+      ? "bg-green-900  text-white border-none"
+      : "bg-red-900  text-white border-none opacity-50"
+    : "bg-violet-200 text-slate-700 border-none";
+
+  return (
+    <div className=" border-b py-2">
+      <h2
+        className="text-xl font-semibold text-slate-700"
+        dangerouslySetInnerHTML={{ __html: props.value.question }}
+      ></h2>
+      <div className="flex gap-2 lg:flex-nowrap flex-wrap my-3">
+        {allAnswer.sort().map((v, k) => {
```

Comment:
You should use descriptive names here instead of `v` and `k` for code readability.